### PR TITLE
Fix: update project accessed at from router and schedulers

### PR DESCRIPTION
**Pull request by lohanidamodar**
## What does this PR do?

- Update project's accessed at from router and schedulers
- For API : https://github.com/appwrite/appwrite/blob/2fb9ebe71df40a258f6c89d8d16a0ba8fc99f331/app/controllers/shared/api.php#L342
- For functions through custom domain https://github.com/appwrite/appwrite/blob/2fb9ebe71df40a258f6c89d8d16a0ba8fc99f331/app/controllers/general.php#L95
- For scheduled executions https://github.com/appwrite/appwrite/blob/2fb9ebe71df40a258f6c89d8d16a0ba8fc99f331/src/Appwrite/Platform/Tasks/ScheduleExecutions.php#L64
- For function scheduler (delayer execution) https://github.com/appwrite/appwrite/blob/2fb9ebe71df40a258f6c89d8d16a0ba8fc99f331/src/Appwrite/Platform/Tasks/ScheduleFunctions.php#L90
- For messaging scheduler https://github.com/appwrite/appwrite/blob/2fb9ebe71df40a258f6c89d8d16a0ba8fc99f331/src/Appwrite/Platform/Tasks/ScheduleMessages.php#L52
- 

## Test Plan

(Write your test plan here. If you changed any code, please provide us with clear instructions on how you verified your changes work. Screenshots may also be helpful.)

## Related PRs and Issues

- (Related PR or issue)

## Checklist

- [x] Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?
- [x] If the PR includes a change to an API's metadata (desc, label, params, etc.), does it also include updated API specs and example docs?

### CODE REVIEW

File: src/Appwrite/Platform/Tasks/ScheduleFunctions.php [hunk 1]
```diff
@@ -7,6 +7,7 @@
 use Utopia\CLI\Console;
 use Utopia\Database\Database;
 use Utopia\Database\DateTime;
+use Utopia\Database\Validator\Authorization;
 use Utopia\Pools\Group;
 
 class ScheduleFunctions extends ScheduleBase
```
Comment (eldadfux) [inline src/Appwrite/Platform/Tasks/ScheduleFunctions.php:13]: Let's move this to the abstraction layer between the two schedulers.

File: src/Appwrite/Platform/Tasks/ScheduleExecutions.php [hunk 2]
```diff
@@ -57,6 +59,16 @@ protected function enqueueResources(Group $pools, Database $dbForPlatform, calla
 
             $delay = $scheduledAt->getTimestamp() - (new \DateTime())->getTimestamp();
 
+            $project = $schedule['project'];
```
Comment (eldadfux) [inline src/Appwrite/Platform/Tasks/ScheduleExecutions.php:62]: What is the difference between this file and the functions scheduler?!

File: app/worker.php [hunk 3]
```diff
@@ -62,7 +62,13 @@
         return $project;
     }
 
-    return $dbForPlatform->getDocument('projects', $project->getId());
+    // NOT all workers need valid DB config so we return empty if this fails
```
Comment (eldadfux) [inline app/worker.php:65]: Do we still need this addition?

File: src/Appwrite/Platform/Tasks/ScheduleExecutions.php (hunk 2 above)
Comment (lohanidamodar) [reply src/Appwrite/Platform/Tasks/ScheduleExecutions.php:62]: I think one is for delayed execution from console and another is for scheduled executions, I don't know exactly

**Review by eldadfux — APPROVED** [review]

**Review by christyjacob4 — APPROVED** [review]

**Pull request merged**